script/coqproject.sh

```diff
@@ -24,24 +24,30 @@ function dep_dirs_lines(){
       namespace_var=${namespace_var//-/_}
       namespace_var=${namespace_var//./_}
       namespace=${!namespace_var:=$2}
-      LINE="-Q $1/$dep_dir/ $namespace"
+      if [ $dep_dir = "." ]; then
```

Comment:
Sure, but the key change is that unless you explicitly pass `$dep_PATH`, `_CoqProject` will not be changed. In other words, `../$dep` is not the default location anymore, `user-contrib' is. Do we want this for all `coqproject`-ized projects?